java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -391,6 +402,24 @@ private string stubMember(Member m) {
   )
 }
 
+private string stubAnnotation(Annotation a) {
+  if exists(a.getAValue())
+  then result = a.toString() + "(" + concat(stubAnnotationValue(a.getAValue()), ",") + ")"
```

Comment:
If more than one annotation element has a value, it is necessary to include the annotation element names, e.g. `@Annotation(value=1, other=2)`

Reply:
Good point, done in 495ce95.